syntax = "proto3";

option java_package = "com.github.klee0kai.proto";

message MetaModel {
  string tag = 1;
  string value = 2;
}

message SomeCmdModel{
  int32 id = 1;
  int64 count = 2;
  float value = 3;
  double valueD = 4;
  string name = 5;
  repeated MetaModel meta = 6;
}


message SomeEventModel{
  int32 id = 1;
  int64 count = 2;
  float value = 3;
  double valueD = 4;
  string name = 5;
  repeated MetaModel meta = 6;
}